modules/thunder_media/thunder_media.install

```diff
@@ -338,3 +339,21 @@ function thunder_media_update_8010() {
 
   return $updateLogger->outputHtml();
 }
+
+/**
+ * Remove auto_select from EB widget config, for those who dont't support it.
```

Comment:
There is typo here "dont't" -> I'm also not sure can we formulate update message to be more understandable for user: fe. Cleanup of "auto_select" option for entity browser widget configurations.
Or something similar, so that it's more clear what update hook is doing. I'm not sure what is best explanation.